src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -1,11 +1,13 @@
-@inject DarkModeService DarkModeService;
-@inject IOptions<AdminOptions> AdminOptions;
+@using OrchardCore.Environment.Shell
```

Comment:
DarkModeService.TenantName so that we can expose this service in liquid later on.

Reply:
I didn't think about a scenario to use it later on in liquid, do you have one?

Reply:
So I've just looked and we have sitename exposed in Liquid.
https://docs.orchardcore.net/en/dev/docs/reference/modules/Liquid/#site

Should we also expose the tenant name then?

Reply:
> I didn't think about a scenario to use it later on in liquid, do you have one?

I did not either when I did the feature. But I figured out afterward that we can create Admin Templates (Liquid) and we can override the current admin Layout.cshtml file from the current Admin theme by using an Admin Template named "Layout". So, yes we should be able to use the Darkmode in Liquid too.

Reply:
And it would clean up that view of that specific `@using OrchardCore.Environment.Shell` which I think should be done in the Service instead.